app/views/external_users/admin/external_users/index.html.haml

```diff
@@ -10,32 +10,34 @@
 = render partial: 'shared/search_form', locals: { search_path: external_users_admin_external_users_path(anchor: 'search-button'), hint_text: t('hint.search_users'), button_text: t('search.users') }
 
 
-= govuk_table_and_caption t('.table_caption', provider_name: "#{current_user.provider.name}"), class: 'govuk-visually-hidden' do
+= govuk_table(class: 'report') do
+  = govuk_table_caption(class: 'govuk-visually-hidden') do
+    = t('.table_caption', provider_name: "#{current_user.provider.name}")
 
-  = govuk_table_thead_collection [t('.last_name'),
+  = govuk_table_thead_collection [ t('.last_name'),
```

Comment:
**nit** not sure why rubocop does not pick this up but generally do not use spaces in within arrays or parentheses

Reply:
Rubocop doesn't runs on haml files (or, more generally, anything other than `*.rb` files).

Reply:
Time for haml linting?